doc/outline.md

```diff
@@ -0,0 +1,36 @@
+documentation outline:
+
+
+- Getting started/Quick Start: 
+	
+	- Explanation of what metric learning is, and what is the purpose of this package
+	- installation
+	- a very quick example on how to import an algo (supervised or not ?) and how to do fit (and predict ?) (and split train and test) on some custom dataset (maybe sklearn.datasets.load_lfw_pairs ?)
+
+- User Guide/List of algorithms: 
+
+	- Supervised Metric Learning: (add links to examples/images from examples at the right place in the description)
+		- Problem setting
+		- Input data (+ see Preprocessor section)
+		- What you can do after fit (transform...)
+
+	- Weakly Supervised Metric Learning: (add links to examples/images from examples at the right place in the description)
+		- Problem setting
+		- Input data (+ See Preprocessor section)
+		- What you can do after fit (predict/score, tranform...)
+		- Scikit-learn compatibility (compatible with grid search + link to example of grid search)
```

Comment:
It might also be good to show a sklearn Pipeline

Reply:
Yes I agree, since the pipeline (Metric Learner, KNN) for instance, represents a major use case of metric learning for classification